account_db_mapper.py

```diff
@@ -0,0 +1,15 @@
+class AccountDatabaseMapper:
+
+    def __init__(self,db):
+        self.db = db
+
+    def save(self,account):
+        self.db.cursor.execute(
+            "INSERT INTO accounts "+
+            "(email,google_access_token,google_refresh_token) " +
+            "VALUES (%s,%s,%s)",
+            (account.email,account.access_token,account.refresh_token)
+        )
+
```

Comment:
Vill vi faktiskt spara ner access_token och refresh_token i databasen då de kommer förmodligen att bytas ut rätt ofta? Jag antar dock att det är bäst att göra det så här for the time being.